test/integration/test-fixtures/image-java-virtualpath-regression/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM alpine:latest@sha256:7144f7bab3d4c2648d7e59409f15ec52a18006a128c733fcff20d3a4a54ba44a
```

Comment:
Same question: will this break if `alpine:latest` moves?

Reply:
Same response for golang - I think we pin to a more specific version - I'm not sure if dockerhub keeps a running log of `latest` or if it updates it in place